sites/public/src/ListingView.tsx

```diff
@@ -176,6 +186,119 @@ export const ListingView = (props: ListingProps) => {
     } else return t("listings.reservedCommunityTitleDefault")
   }
 
+  const hasMethod = (applicationMethods: ApplicationMethod[], type: ApplicationMethodType) => {
+    return applicationMethods.some((method) => method.type == type)
+  }
+
+  const getMethod = (applicationMethods: ApplicationMethod[], type: ApplicationMethodType) => {
+    return applicationMethods.find((method) => method.type == type)
+  }
+
+  type AddressLocation = "dropOff" | "pickUp"
+
+  const getAddress = (
+    addressType: ListingApplicationAddressType | undefined,
+    location: AddressLocation
+  ) => {
+    if (addressType === ListingApplicationAddressType.leasingAgent) {
+      return listing.leasingAgentAddress
+    }
+    if (addressType === ListingApplicationAddressType.mailingAddress) {
+      return listing.applicationMailingAddress
+    }
+    if (location === "dropOff") {
+      return listing.applicationDropOffAddress
+    } else {
+      return listing.applicationPickUpAddress
+    }
+  }
+
+  const cloudinaryPdfFromId = (publicId: string, cloudName: string) => {
+    return `https://res.cloudinary.com/${cloudName}/image/upload/${publicId}.pdf`
+  }
+
+  const getOnlineApplicationURL = () => {
+    let onlineApplicationURL
+    if (hasMethod(listing.applicationMethods, ApplicationMethodType.Internal)) {
+      onlineApplicationURL = `/applications/start/choose-language?listingId=${listing.id}`
+    } else if (hasMethod(listing.applicationMethods, ApplicationMethodType.ExternalLink)) {
+      onlineApplicationURL =
+        getMethod(listing.applicationMethods, ApplicationMethodType.ExternalLink)
+          ?.externalReference || ""
+    }
+    return onlineApplicationURL
+  }
+
+  const getPaperApplications = () => {
+    return (
+      getMethod(listing.applicationMethods, ApplicationMethodType.FileDownload)
+        ?.paperApplications.sort((a, b) => {
+          // Ensure English is always first
+          if (a.language == "en") return -1
+          if (b.language == "en") return 1
+
+          // Otherwise, do regular string sort
+          const aLang = t(`languages.${a.language}`)
+          const bLang = t(`languages.${b.language}`)
+          if (aLang < bLang) return -1
+          if (aLang > bLang) return 1
+          return 0
+        })
+        .map((paperApp) => {
+          return {
+            fileURL: paperApp?.file?.fileId.includes("https")
+              ? paperApp?.file?.fileId
+              : cloudinaryPdfFromId(
+                  paperApp?.file?.fileId || "",
+                  process.env.cloudinaryCloudName || ""
+                ),
+            languageString: t(`languages.${paperApp.language}`),
+          }
+        }) ?? null
+    )
+  }
+
+  const applySidebar = () => (
```

Comment:
This replaces `Application Section`, and `Applications` and `ApplicationAddresses` replace `Apply